src/doc/dev_manual/LiveCustomerResponse.rst

```diff
@@ -141,34 +156,36 @@ part of an escalation. To the extent possible, temporary delegation from
-significant *escalations* (see below). Thus, other developers are free to ignore
-customer inquiries as well as redirect customers who may contact them directly
+significant :ref:`escalations <sre_escalations>`. Thus, other developers are free to
+ignore customer inquiries as well as redirect customers who may contact them directly
 via email, phone or walk-in. It is a best practice to handle such redirections
 with a formal, three-way handoff confirming that the customer indeed makes
 contact with the **Primary**.
 
-Active Customer Response Issues Repo
-------------------------------------
+.. _sre_active_issues:
 
-*Active* customer response issues will be logged and tracked in a separate GitHub,
+Active SRE Issues Repo
+----------------------
+
+*Active* SRE issues will be logged and tracked in a separate GitHub,
 `issues-only repository <https://github.com/visit-dav/live-customer-response/issues>`_
 within the `visit-dav GitHub organization <https://github.com/visit-dav>`_. 
 For each new inquiry, the primary will file an issue ticket and assign themselves.
-When the incident is resolved, the associated issue is closed. The primary will
+When the inquiry is resolved, the associated issue is closed. The primary will
 endeavor to capture all relevant information and communications in this issue.
 The use of GitHub issues for this purpose has a number of advantages over other
-options such as email including better support for attachments. For this reason,
-a number of steps were taken to integrate the ``visit-users@elist.ornl.gov`` email
-list with this issues-only repository.
-
-Upon receiving a *new* inquiry on the ``visit-users`` email list (or a telephone
-hotline call or walk-in with the exception of those involving classified information),
-the procedure is for the **Primary** to cut-n-paste the initial email to a new GitHub
-`Customer Response issue <https://github.com/visit-dav/live-customer-response/issues>`_
-and from then on handle all communication
-through the *conversation* associated with that issue. Each comment there
+options such as email including better search/browse as well as support for
+attachments. For this reason, a number of steps were taken to integrate the
+``visit-users@elist.ornl.gov`` email list with this issues-only repository.
+
+Upon receiving a *new* inquiry on the ``visit-users`` email list, telephone hotline
+call, or walk-in (with the exception of walk-ins involving classified information),
+the procedure is for the **Primary** to cut-n-paste the initial email to a new
+GitHub `SRE issue <https://github.com/visit-dav/live-customer-response/issues>`_
+and from then on handle all communication through the *conversation* associated with
+that issue. Each comment there
 will generate an email to ``visit-users``. In addition, any reply to any
 GitHub generated emails will result in a new comment added to the GitHub issues
 as though it came from GitHub user account ``markcmiller86-visit``. However,
```

Comment:
Is it possible to create a github user called something like visit-support and use that account instead of markcmiller86-visit?

Reply:
> Is it possible to create a github user called something like visit-support and use that account instead of markcmiller86-visit?

Yes, but is currently logistically problematic. I will put on the todo list though. I should have thought to call it that to begin with 😕